docs/source/tutorials/06_using_your_own_models.rst

```diff
@@ -406,8 +410,8 @@ node.
                }
 
 The custom node takes in the in-built PeekingDuck :term:`img` data type, makes a prediction based
```

Comment:
Another instance of "in-built" instead of "built-in"

Reply:
Replaced